src/SearchField/README.md

```diff
@@ -0,0 +1,88 @@
+# SearchField
+
+Provides a basic search field component.
+
+## API
+
+### `screenReaderText` (string; required)
+`screenReaderText` specifies the text for screen readers within the `label` element of the `InputText` component.
+
+### `onSubmit` (function; required)
+`onSubmit` specifies a callback function for when the `SearchField` is submitted by the user. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onSubmit={(value) => { console.log(value); }}
+/>
+```
+
+### `onChange` (function; optional)
+`onChange` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onChange={(value) => { console.log(value); }}
+/>
+```
+
+### `onFocus` (function; optional)
+`onFocus` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
+
+```jsx
+<SearchField
+  screenReaderText="search"
+  onFocus={(value) => { console.log(value); }}
+/>
+```
+
+### `onBlur` (function; optional)
+`onBlur` specifies a callback function for when the value in `SearchField` is changed by the user. The default is an empty function. For example:
```

Comment:
Is the callback executed when the value is changed or when the `SearchField` loses focus?